pages/wiki/watchfaces-creation.md

```diff
@@ -3,7 +3,7 @@ title: Watchfaces Creation
 layout: documentation
 ---
 
-<p>Watchface creation is fairly simple. The provided scripts make it possible to create your first watchface even without any programming skills by redesigning existing SVG images only. Changing code requires some QML knowledge. Even if you are not familiar with QML yet, prior knowledge of any scripting language should be sufficiant to learn quickly from the already existing examples provided by the community.<br>Those community examples are collected in the <a href="https://github.com/AsteroidOS/unofficial-watchfaces">unofficial-watchfaces repo</a>. This guide will walk you through the steps of how to get started.</p>
+<p>Our goal is to make watchface creation fairly simple.<br>The provided scripts make it possible to create your first watchface even without any programming skills by redesigning existing SVG images.<br>Changing code requires some QML knowledge. Even if you are not familiar with QML yet, prior knowledge of any scripting language should be sufficiant to learn quickly from the already existing examples provided by the community.<br>Those community examples are collected in the <a href="https://github.com/AsteroidOS/unofficial-watchfaces">unofficial-watchfaces repo</a>. This guide will walk you through the steps of how to get started.</p>
```

Comment:
Eheh, I find it funny how you went from "Watchface creation is fairly simple" to "Our goal is to make watchface creation fairly simple"

Reply:
Honesty wins.
And the "Is fairly simple" is/was obviously form your PoV :D
But what i see is that it gets easier and easier. So we can claim that as our goal easily. Which it is on my side.